### PR TITLE
Add data in profile about user

### PR DESCRIPTION
**Pull request by Shutya**

### CODE REVIEW

File: src/containers/ProfileContainer/ProfileContainer.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import React, {Component} from 'react';
+import _ from 'lodash';
```
Comment (SergeyKovalchuk) [inline src/containers/ProfileContainer/ProfileContainer.js]: Please add https://www.npmjs.com/package/babel-plugin-lodash to webpack config for such usage

Comment (Shutya) [reply src/containers/ProfileContainer/ProfileContainer.js]: Done)

File: src/containers/ProfileContainer/ProfileContainer.js [hunk 2]
```diff
@@ -14,9 +20,49 @@ import style from './style.scss';
...
+  }
+
+  onSubmitProfileDetails = (e) => {
+    e.preventDefault();
+    this.props.userEdit({id: this.props.userInfo._id, ...this.state.profileDetails});
   }
 
   render() {
+    const {userInfo} = this.props;
+    if (!userInfo) {
```
Comment (SergeyKovalchuk) [inline src/containers/ProfileContainer/ProfileContainer.js]: Please move down this part and put it in other hook

Comment (Shutya) [reply src/containers/ProfileContainer/ProfileContainer.js]: Done

**Pull request merged**